libs/barista-components/sunburst-chart/src/sunburst-chart.ts

```diff
@@ -288,4 +293,11 @@ export class DtSunburstChart {
       };
     }
   }
+
+  /**
+   * Gets a safe CSS
+   */
```

Comment:
```suggestion
  /**
   * Sanitization of the custom property is necessary as, custom property assignments do not work
   * in a viewEngine setup. This can be removed with angular version 10, if ivy is no longer opt out. 
  */
```